### PR TITLE
Solution

### PR DESCRIPTION
**Pull request by Vasyl-Poremchuk**

### CODE REVIEW

**Review by VitaliyBychevoy — APPROVED** [review]

File: app/create_file.py [hunk 1]
```diff
@@ -1 +1,30 @@
-# write your code here
+import os
+import sys
+from datetime import datetime
+
+
+def write_contents_to_file(filename: str) -> None:
+    with open(filename, "a") as file:
+        file.write(datetime.now().strftime("%Y-%m-%d %H:%M:%S"))
```
[the remark this answers is not in the document]
Comment (Nattalli) [reply app/create_file.py:8]: In the examples of the task when inserting new text, first comes the empty line, and then the new values

File: app/create_file.py [hunk 2]
```diff
@@ -1 +1,30 @@
...
+        while (line_content := input("Enter content line: ")) != "stop":
+            file.write(f"{line_number} {line_content}\n")
+            line_number += 1
+
+
+def create_file() -> None:
+    input_command = sys.argv
+
+    if "-d" in input_command and "-f" in input_command:
+        path = os.path.join(*input_command[2:-2])
```
Comment (Nattalli) [inline app/create_file.py:21]: wrong realization
trying this `python create_file.py -d dir5` will do this:
Снимок экрана 2022-10-24 в 12 05 36

**Review by Nattalli — CHANGES_REQUESTED** [review]

File: app/create_file.py (hunk 2 above)
Comment (Vasyl-Poremchuk) [reply app/create_file.py]: Now it should work correctly)

**Review by Nattalli — CHANGES_REQUESTED** [review]
Change one more thing

File: app/create_file.py [hunk 3]
```diff
@@ -1 +1,39 @@
...
+from datetime import datetime
+
+
+def write_contents_to_file(filename: str) -> None:
+    with open(filename, "a") as file:
+        file.write(datetime.now().strftime("%Y-%m-%d %H:%M:%S") + "\n")
+
+        line_number = 1
+
+        while (line_content := input("Enter content line: ")) != "stop":
```
[the remark this answers is not in the document]
Comment (Vasyl-Poremchuk) [reply app/create_file.py:12]: Oops, I didn't notice that :)

**Review by Nattalli — APPROVED** [review]
Good job!

**Pull request closed without merging**